authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+
+from .models import Article
+
+
+class ArticleSerializer(serializers.ModelSerializer):
```

Comment:
@KabohaJeanMark Some of the fields are required as bare minimum for an article. Do you cater for this anywhere. I don't see any validation for such.

####Screenshot
<img width="486" alt="Screen Shot 2019-04-29 at 10 56 27" src="https://user-images.githubusercontent.com/36948493/56882900-bdcfc300-6a6d-11e9-8808-00f081ceced4.png">